### ISSUE TITLE
.NET 6: Verify if the IDE needs updates to support Mac Catalyst asset resources

### ISSUE THREAD

**Issue by rolfbjarne** [xamarin/xamarin-macios#10818]
Asset resources are different for Mac Catalyst than for iOS.

We must verify whether the IDEs need updates, and if so, implement those updates.

PR that shows how asset catalog resources for Mac Catalyst: https://github.com/xamarin/xamarin-macios/pull/10815/files

**Comment by rolfbjarne** [comment]
@emaf is this something your team could look at for the IDE(s)?

**Comment by emaf** [comment]
@rolfbjarne Yes! We'll look into this as part of the effort to support the Assets editor on .NET 6 projects.

**Comment by rolfbjarne** [comment]
@emaf has this been done?

**Comment by emaf** [comment]
@rolfbjarne No, this is still pending

**Comment by rolfbjarne** [comment]
Moved to .NET 7.

**Comment by emaf** [comment]
I'm closing this one because the assets editor has been deprecated in VSM, and instead it will open Xcode.
